client/state/comments/selectors/get-parent-comment.js

```diff
@@ -19,8 +16,8 @@ import { getPostCommentsTree } from 'state/comments/selectors';
  */
 export const getParentComment = ( state, siteId, postId, commentId ) => {
 	const commentsTree = getPostCommentsTree( state, siteId, postId, 'all' );
-	const parentCommentId = get( commentsTree, [ commentId, 'data', 'parent', 'ID' ], 0 );
-	return get( commentsTree, [ parentCommentId, 'data' ], {} );
+	const parentCommentId = commentsTree[ commentId ]?.data.parent?.ID ?? 0;
+	return commentsTree[ parentCommentId ]?.data ?? {};
 };
 
 export default getParentComment;
```

Comment:
Feels so weird to export this both as a named export and a default export 🤔 Should we just go with the default one?

Reply:
There are a lot of files in the codebase that follow this pattern, for some reason 🤷‍♂

I've removed the default export instead, for consistency with the rest of the imported files in `state/comments`, but I'm happy to switch it around to removing the named export instead if you think that's best.

Reply:
Yeah, I just noticed that 🤔 I don't see a good reason for it, to be honest.

And yes, if a module exposes a single export, I don't see a reason to go with a named export; I'd prefer a default export in that case. Unless you convince me there is any sort of improvement that the named export brings 😉 But if it did, why would we ever want to use default exports? 😉

Reply:
I'm not a huge fan of default exports, to be honest 😅 I'm not quite sure why they exist in an ES module world. Presumably, it's to simplify the case where only one thing is being exported from a module. But do they?

```js
import foo from './foo';
```

```js
import { foo } from './foo';
```

You still need to name your import if you want to use it.

And they're actually worse with dynamic imports:

```js
const foo = ( await import( './foo' ) ).default;
```

```js
const { foo } = await import( './foo' );
```

The only situation where it is slightly easier is if you want to use a particular name for something:

```js
import theNameIWantToUse from './foo';
```

```js
import { foo as theNameIWantToUse } from './foo';
```

But then again, that can also make re-exporting harder:

```js
export { default as foo } from './foo';
```

```js
export { foo } from './foo';
```

All in all, I think it just leads to confusion, particularly when a lot of devs get introduced to imports and exports through frameworks like React, which export the same things as both named exports and default exports 😞 

That said, contrary to what the above may lead you to think, my feelings on this matter aren't particularly strong 😄 I'm happy enough to go to default exports, but it's probably best to maintain consistency. Should I go ahead and use default exports on all the selector modules?

Reply:
Thanks for the thorough explanation 👍 

 I don't think my feelings towards using default exports are that strong; and even if they were, we could do that in a subsequent PR. This one isn't too small anyway 😉 

Let's keep the named ones for the time being. 👍